index.html

```diff
@@ -1,21 +1,142 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+
     <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="/css/main.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300&display=swap"
+      rel="stylesheet"
+    />
 
     <title>Responsive grid project</title>
-</head>
+  </head>
+
+  <body>
+    <header>
+      <div class="toggle-menu">
+        <hr class="line-one" />
+        <hr class="line-two" />
+        <hr class="line-three" />
+      </div>
+
+      <div class="blue-box"></div>
+      <h1>Maya Nelson</h1>
```

Comment:
The `h1` should be the text that best describes this page, as distinct from the others in the site. So if all the pages are about Maya Nelson, then the text "Maya Nelson" won't be the best choice. Which heading on this page best describes what this page is about? That should be the h1.